### PR TITLE
Add extra docs for azure

### PR DESCRIPTION
**Pull request by SomtochiAma**
Signed-off-by: Somtochi Onyekwere <somtochionyekwere@gmail.com>

### CODE REVIEW

File: content/en/docs/guides/image-update.md [hunk 1]
```diff
@@ -724,6 +724,19 @@ patches:
     - op: add
       path: /spec/template/spec/containers/0/args/-
       value: --azure-autologin-for-acr
-```
+# Add this if you are using aad pod identity with managed identity (and remember to remove this line :))
```
Comment (stefanprodan) [inline content/en/docs/guides/image-update.md:727]: Comments are harmless, I don’t think we need to tell people to remove them.

Comment (SomtochiAma) [reply content/en/docs/guides/image-update.md:727]: Running Kustomize build errors out because of it

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @SomtochiAma

**Comment by wiebeck** [comment]
Thanks for updating the docs! Much appreciated. :)

**Pull request merged**